### PR TITLE
Fix filename filter in flywheel View [GEAR-251]

### PR DESCRIPTION
**Pull request by NPann**
Fix GEAR-251

View now return any file type.

### CODE REVIEW

File: transfer_log.py [hunk 1]
```diff
@@ -300,8 +300,8 @@ def get_hierarchy(client, config, project_id, case_insensitive=False):
         columns.append('session.timezone')
     if container_type == 'acquisition':
         view = client.View(columns=columns, container=container_type,
-                       filename='*.zip', process_files=False, match='all',
-                       sort=False)
+                           filename='*', process_files=False, match='all',
```
Comment (kalebdfischer) [inline transfer_log.py]: Might be worthwhile to expand the template to have a field that is placed here and default to *.zip so the change is non-breaking. Or we can break and increment major version.

File: manifest.json [hunk 2]
```diff
@@ -2,11 +2,11 @@
   "name": "transfer-log-report",
   "label": "GRP-5: Transfer Log Report Validation",
   "description": "Validate existence of data referenced within a transfer report log within a given project. This Gear outputs a csv file with the fields: 'error', 'path',  'type', 'resolved', 'label', '_id'.",
-  "version": "0.3.4",
+  "version": "0.3.5",
```
Comment (kalebdfischer) [inline manifest.json]: As is, this is a breaking change, so the version increment should be incremented unless you decide to revert to a default of *.zip

File: transfer_log.py (hunk 1 above)
Comment (NPann) [reply transfer_log.py]: Great idea! Done in ad066d5

File: manifest.json (hunk 2 above)
Comment (NPann) [reply manifest.json]: Bump to 0.4.0 given new filename field in template

**Comment by NPann** [comment]
All changes addressed. Confirmed that it runs fine locally.

**Pull request merged**